packages/web/src/modules/zoom/requests.ts

```diff
@@ -0,0 +1,39 @@
+import { services } from '@application';
+import {
+  DataResponse,
+  Endpoint,
+  RequestFetch,
+  RequestPost,
+} from '@chess-tent/types';
+
+interface ZoomAuthorizeData {
+  refresh_token: string;
+  access_token: string;
+  token_type: string;
+}
+
+type ZoomAuthorizeRequest = RequestFetch<
+  Endpoint<
```

Comment:
All endpoint types are defined in `types/api` and then decorated in `web/application/types/endpoints`. Please take a look at that pattern, we can discuss it.